packages/integration/package.json

```diff
@@ -11,17 +11,18 @@
     "@types/stubby": "^4.1.0",
     "const-max-int32": "^1.0.2",
     "express": "^4.17.1",
+    "get-port": "^7.1.0",
     "js-yaml": "^3.14.0",
     "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.20",
     "serve-handler": "^6.1.3",
     "server": "*",
     "stubby": "^5.0.0",
-    "ts-node": "^9.0.0",
-    "typescript": "^4.0.3"
```

Comment:
Any particular reason we need to use a fixed version of typescript?

Reply:
you're right, have updated to allow variable version